server/filestore.go

```diff
@@ -547,22 +547,22 @@ func (fs *fileStore) enableLastMsgBlockForWriting() error {
 }
 
 // Store stores a message.
-func (fs *fileStore) StoreMsg(subj string, msg []byte) (uint64, error) {
+func (fs *fileStore) StoreMsg(subj string, msg []byte) (seq uint64, ts int64, err error) {
```

Comment:
Again in general I avoid this pattern even though its clear due to aliasing bugs.

Reply:
Also why do we need ts on store? I can see on load etc.

Reply:
it's used to rapidly access the ts in processInboundJetStreamMsg() without having to do any fetching of anything

Reply:
Ah ok so for online users who we can directly deliver to, gotcha..